docs/COMPONENTS.md

```diff
@@ -21,57 +24,61 @@ Components use the following definition:
+    // and paragraphs.
     // Default = false.
     "countStatistics": true | false,
 
-    // Allows nesting the component in containers.
-    // Defaults = yes.
+    // Allow nesting the component in containers.
+    // Default = yes.
     "allowNesting": "yes" | "no" | "one-level",
 
-    // Restricts children of this component to the listed ones.
-    // The "withContent" key can be used to filter down on the content of a directive.
+    // Define which types of components can be made a child of another component.
+    // Use the "withContent" key to filter down on the content of a directive.
     // For example this can be used to require that a doc-image directive has an image
     // applied by the user.
     "restrictChildren": {
         "image-comp": { "withContent": "image" }
     },
 
-    // Optional property which overrides the global "defaultComponentOnEnter" property.
-    // For example it can be used to define a special container component where the default component which
-    // is created when pressing Enter will differ from the default component that is added when pressing
-    // Enter outside the container.
-    // Introduced in version 1.1.0
+    // Override the default component that is added when pressing Enter (as defined in the global 
+    // "defaultComponentOnEnter" property).
+    // This can be useful for example in a Container when this behavior should be different compared to the main story.
+    // Introduced in version 1.1.0.
     "defaultComponentOnEnter": "componentName"
 
-    // Optional property to configure options per directive in the component
+    // Configure options per directive
     "directiveOptions": {
         "directiveKey": {
-            // Optional property which defines directives autofill rules.
-            // It can be used when the content of a component should be filled automatically.
-            // For example the editable directive can be filled from the image directive when an image is added to an article,
-            // using one of metadata properties of the Enterprise image object (it currently works with Enterprise metadata only)
-            // There is also an ability to set different trigger options. "Once" means that it will be triggered on first
-            // data setting, in case of image directive - when image is added for the first time
-            // Introduced in version 1.1.0
-            // Version 1.2.0 deprecated the trigger option
+            //
+            // Autofill
+            // (Optional) Define if the content of a component should be filled automatically.
+            // For example: automatically fill the caption of an image when an image is added to an article.
+            // Currently only works with Enterprise metadata only.
+            //
+            // Set different trigger options: 
+            // (Introduced in version 1.1.0, deprecated since version 1.2.0)
```

Comment:
Not sure this communicates well enough that the autofill+trigger option was added to 1.1.0 and that just the trigger has been removed in 1.2.0